src/tools/compiletest/src/runtest.rs

```diff
@@ -3290,7 +3250,9 @@ impl ProcRes {
              {}\n\
              ------------------------------------------\n\
              \n",
-            self.status, self.cmdline, self.stdout, self.stderr
+            self.status, self.cmdline,
+            json::extract_rendered(&self.stdout),
+            json::extract_rendered(&self.stderr),
```

Comment:
I assume this is the bulk of the changes? :tada:

Reply:
jop